helm/values.yaml

```diff
@@ -176,6 +176,11 @@ namespacedValidation:
 automaticChildApproval:
```

Comment:
Unrelated note: This still experimental?

Reply:
:shrug:  was it ever really experimental?